src/site/confluence/index.confluence

```diff
@@ -4,6 +4,10 @@ h2. What is Curator?
 
 Curator _n &#x02c8;kyoor&#x035d;&#x02cc;&#x0101;t&#x0259;r_: a keeper or custodian of a museum or other collection \- A ZooKeeper Keeper.
 
+Then, what is Apache Curator? Apache Curator is a Java/JVM client library for [[Apache ZooKeeper|https://zookeeper.apache.org/]], a distributed coordination service.
```

Comment:
"Then, what is Apache Curator?" is unnecessary IMO. I prefer the line without this.